scene/gui/color_picker.cpp

```diff
@@ -855,6 +889,14 @@ ColorPicker::ColorPicker() :
 
 /////////////////
 
+void ColorPickerButton::_about_to_popup() {
+	set_pressed(true);
+	// Displaying the old color only makes sense for ColorPickerButton
+	// (as it can be closed and reopened), not standlone ColorPickers.
+	picker->set_display_old_color(true);
```

Comment:
You can use `picker->set_display_old_color(true);` inside `_update_picker()` as it's needed only once.